### ISSUE TITLE
errors window does not display score error list

### ISSUE THREAD

**Issue by Bacchushlg** [Audiveris/audiveris#56]
The "old" handbook describes that errors are listed in the error window (chapter glyph merge).
I don't get any entries here, although I have errors after analysis.
Is the feature obsolete now? The idea as described in the handbook is not bad...

**Comment by hbitteur** [comment]
Yes, it was a V4 feature. Perhaps it could be resurrected for v5. I need to check its feasibility because the model is now very different...
